src/08_ecosystem/00_chapter.md

```diff
@@ -0,0 +1,63 @@
+# The Async Ecosystem
+Most Rust asynchronous projects will need crates provided by the community to function.
+The [`futures` crate](https://docs.rs/futures/) contains traits and functions useful for writing async code.
+In addition, there are several crates that provide async runtimes,
+which are used to run asynchronous functions from synchronous code.
+This section introduces the crates most commonly used in async projects.
+
+## Asynchronous Runtimes
+Async runtimes are libraries that allow async functions to be run from sync functions, including the `main` function.
```

Comment:
> libraries that allow async functions to be run from sync functions

While true, it could be clearer to say "libraries that execute async applications"